cmd/commands/runtime.go

```diff
@@ -819,40 +829,48 @@ func NewRuntimeUninstallCommand() *cobra.Command {
 }
 
 func RunRuntimeUninstall(ctx context.Context, opts *RuntimeUninstallOptions) error {
+	defer printSummaryToUser()
+
 	// check whether the runtime exists
 	if !opts.SkipChecks {
 		_, err := cfConfig.NewClient().V2().Runtime().Get(ctx, opts.RuntimeName)
+		appendLogToSummary("Checking if runtime exists", err)
 		if err != nil {
-			log.G(ctx).Warn("you can attempt to uninstall again with the \"--skip-checks\" flag")
+			summaryArr = append(summaryArr, summaryLog{"you can attempt to uninstall again with the \"--skip-checks\" flag", Info})
 			return err
 		}
 	}
 
 	log.G(ctx).Infof("Uninstalling runtime '%s'", opts.RuntimeName)
 
-	if err := apcmd.RunRepoUninstall(ctx, &apcmd.RepoUninstallOptions{
+	err := apcmd.RunRepoUninstall(ctx, &apcmd.RepoUninstallOptions{
 		Namespace:    opts.RuntimeName,
 		Timeout:      opts.Timeout,
 		CloneOptions: opts.CloneOpts,
 		KubeFactory:  opts.KubeFactory,
 		Force:        opts.Force,
 		FastExit:     opts.FastExit,
-	}); err != nil {
+	})
+	appendLogToSummary("Uninstalling repo", err)
+	if err != nil {
 		if !opts.Force {
-			log.G(ctx).Warn("you can attempt to uninstall again with the \"--force\" flag")
+			summaryArr = append(summaryArr, summaryLog{"you can attempt to uninstall again with the \"--force\" flag", Info})
```

Comment:
why do we somtimes use this to append
```
summaryArr = append(summaryArr, summaryLog{"...
```
and sometimes this:
```
appendLogToSummary("Uninstalling repo", err)
```
?

Reply:
because the appendLogToSummary is appending with `Success` or `Failure` according to the err, and if it's `Info` level, I append it with append.